### PR TITLE
Fixing a couple typos in Contributing document

### PR DESCRIPTION
**Pull request by kenrachynski**
It's possible that not everybody will already know how to set everything up for package development.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,15 +13,17 @@ branch only.
...
 To build development versions, first you need a configuration file in
-`~\sublime-package-dev.json`.
+`~\.sublime-package-dev.json`.
 
 This file must contain the following information:
 
     {
-        "pathToSublimeText": "...\sublime_text.exe",
-        "pathToSublimeTextData": "...\
+        "pathToSublimeText": "...\\sublime_text.exe",
```
Comment (vors) [inline CONTRIBUTING.md]: Is it right that you need `\\` instead of `\`?

Comment (davidobrien1985) [inline CONTRIBUTING.md]: If it's compiled on Linux / Mac, yes, that should be alright.

Comment (vors) [inline CONTRIBUTING.md]: Oh, I see. I tested it on Windows and I think it worked with a single back-slash.

Comment (kenrachynski) [inline CONTRIBUTING.md]: I was trying to use this on Windows and it didn't work until I switched it to double back-slash.

**Pull request merged**